src/components/Dialog.tsx

```diff
@@ -111,7 +111,7 @@ export const Modal = styled.div<{ color: Color }>`
   position: absolute;
   top: 0;
   width: 100%;
-  z-index: ${Layer.DIALOG};
+  z-index: 1;
```

Comment:
do we want to add a layer and if so what should we call it? If this is left as Dialog, it has overlap seen in the attached image